packages/preset-panda/src/semanticTokens.ts

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { defineSemanticTokens } from "@pandacss/dev";
+
+export const semanticTokens = defineSemanticTokens({
+  spacing: {
+    "4xsmall": { value: "{spacing.2}" },
+    "3xsmall": { value: "{spacing.3}" },
+    xxsmall: { value: "{spacing.4}" },
+    xsmall: { value: "{spacing.6}" },
+    small: { value: "{spacing.8}" },
+    medium: { value: "{spacing.12}" },
+    large: { value: "{spacing.15}" },
+    xlarge: { value: "{spacing.18}" },
+    xxlarge: { value: "{spacing.24}" },
+    "3xlarge": { value: "{spacing.36}" },
+    "4xlarge": { value: "{spacing.48}" },
```

Comment:
Vi har gått ordentlig inn på t-skjorte-størrelser. Prøver alltid å sørge for at "medium" er midtpunktet.